x-pack/plugins/infra/public/pages/logs/stream/page_view_log_in_context.tsx

```diff
@@ -84,6 +84,7 @@ const LogInContextWrapper = euiStyled.div<{ width: number | string; height: numb
   padding: 16px;
   width: ${(props) => (typeof props.width === 'number' ? `${props.width}px` : props.width)};
   height: ${(props) => (typeof props.height === 'number' ? `${props.height}px` : props.height)};
+  max-height: 75vh; // Same as EuiModal
```

Comment:
Do we want to hard-code it like this even though there's a parameterized `height` prop? According to the CSS spec "`max-height` overrides `height`", so I wonder if we could somehow unify the treatment of both.

Reply:
> According to the CSS spec "`max-height` overrides `height`"

I'm relying on this :) I want the content of the modal to "take as much height available (the `height`) without making it bigger than its container (the `max-height`, which replicates the value in the `EuiModal`).

The problem is that we have no control on the height of the modal itself. The content can be as tall as we want, but the modal will clip it no matter what *.

---

*: in Chrome at least. In Firefox it worked correctly, but I'm not sure what browser is breaking the standards here.